README.md

```diff
@@ -423,6 +423,15 @@ Note that the public directory name is not included in the URL. A file
 Use the `:static_cache_control` setting (see [below](#cache-control)) to add
 `Cache-Control` header info.
 
+By default, Sinatra serves static files from the `public/` folder without running middleware or filters. To add custom headers (e.g, for CORS or caching), use the `:static_headers` setting:
+
+```ruby
+  set :static_headers, {
+    'Access-Control-Allow-Origin' => '*',
+    'X-Static-Asset' => 'served-by-sinatra'
+  }
+```
+
```

Comment:
We should also document the new setting under "Available Settings" in the README, preferably after `static_cache_control`